packages/core/aqua-to-js/src/generate/interfaces.ts

```diff
@@ -117,12 +119,19 @@ export class TSTypeGenerator implements TypeGenerator {
     const serviceDecl = `service: ${srvName}Def`;
     const serviceIdDecl = `serviceId: string`;
 
-    const registerServiceArgs = [
-      [serviceDecl],
-      [serviceIdDecl, serviceDecl],
-      [peerDecl, serviceDecl],
-      [peerDecl, serviceIdDecl, serviceDecl],
-    ];
+    const registerServiceArgs =
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      (srvDef.defaultServiceId as DefaultServiceId).s_Some__f_value != null
```

Comment:
this looks pretty hacky to be honest. Sometimes defaultServiceId is not a string and has some weird property in it? I would avoid this, should be possible somehow

Reply:
I dunno how to avoid :)
It's actual value that comes from aqua compiler

Reply:
We can discuss this someday. I created a [task](https://linear.app/fluence/issue/DXJ-531/strange-type-from-aqua-compiler)

Reply:
Because these types are generated from compiler team :)

Reply:
It's 1 more question to ask how to split these types. I'm feeling ok to put them in `aqia-api`.